backend/src/helpers/helpers.js

```diff
@@ -0,0 +1,41 @@
+// Priorities are set from the lowest to the highest.
+const statePriorities = [ 'away', 'viewing', 'commenting', 'editing', 'merging' ];
+
+module.exports = { getUserListFromSessions };
```

Comment:
Please use one convention for each file. E.g. in SessionRepository `module.exports` is at the end of the file.

Reply:
Fixed.